docs/analytics_tools/python_libraries.md

```diff
@@ -86,7 +86,7 @@ from calitp_data_analysis.sql import query_sql
 df_dim_feeds = query_sql("""
 SELECT
     *
-FROM `views.gtfs_schedule_dim_feeds`
+FROM `mart_gtfs.dim_agency`
```

Comment:
nit: I'd have the name of the dataframe align with the table that's queried, so I'd either query `dim_schedule_feeds` or change the name of the dataframe to `df_dim_agency`

Reply:
Resolved in [e6f2773](https://github.com/cal-itp/data-infra/pull/2602/commits/e6f2773037116a38d5e1b6783fa7c63729318178)